org.seventhstring.Transcribe.yml

```diff
@@ -0,0 +1,40 @@
+app-id: org.seventhstring.Transcribe
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: transcribe
+finish-args:
+#TODO: figure out if its possible to only grant permissions 
+# on ~/.Transcribe!7 and xdg-music:ro, for now we use home
+# as a functional compromise
+- --share=ipc
+- --socket=x11
+- --socket=wayland
+- --socket=pulseaudio
+- --filesystem=home
+modules:
+  - name: transcribe
+    buildsystem: simple
+    build-commands: 
+    - mkdir -p /app/bin
+    - cp transcribe libgstvideosection.so xschelp.htb /app/bin
+    - mkdir -p /app/share/icons/hicolor/16x16/apps && mv gtkicons/xsc16x16.png /app/share/icons/hicolor/16x16/apps/org.seventhstring.Transcribe.png
+    - mkdir -p /app/share/icons/hicolor/32x32/apps && mv gtkicons/xsc32x32.png /app/share/icons/hicolor/32x32/apps/org.seventhstring.Transcribe.png
+    - mkdir -p /app/share/icons/hicolor/48x48/apps && mv gtkicons/xsc48x48.png /app/share/icons/hicolor/48x48/apps/org.seventhstring.Transcribe.png
+    - mkdir -p /app/share/icons/hicolor/128x128/apps && mv gtkicons/xsc128x128.png /app/share/icons/hicolor/128x128/apps/org.seventhstring.Transcribe.png
+    - |
```

Comment:
Why don't you run the install script instead?

Reply:
I initially tried to and then rename the icon/.desktop file using the `rename-icon` and `rename-desktop-file` option but that did not work. Also running the install script requires `xdg-utils` to be installed and it seemed simpler just to copy the files.

Having said that, should i try again using the install script?

Reply:
ok i figured out how to make `rename-icon` and `rename-desktop-file` work. now only the install script is called